scripts/imm

```diff
@@ -145,16 +147,24 @@ case "$OPERATION" in
 				[[ -z ${PARAM_1} ]] && read -p "Please provide endpoint name " PARAM_1
 				[[ -z ${PARAM_2} ]] && read -p "Please provide model name " PARAM_2
 				[[ -z ${PARAM_3} ]] && read -p "Please provide model version policy (press enter for default)" PARAM_3
-				[[ -z ${PARAM_4} ]] && read -p "Please provide tenant name " PARAM_4
+				[[ -z ${PARAM_4} ]] && read -p "Please provide tenant name (press enter for default) " PARAM_4
```

Comment:
`if [[ -z ${DEFAULT_TENANT_NAME} ]]; then` use this before that line
the same thing in all cases